daffodil/parser.py

```diff
@@ -58,7 +58,7 @@ class ArrayEnd(Token): pass
 
 class DaffodilParser(object):
     def __init__(self, src):
-        self.src = "{" + src + "}"
+        self.src = "{" + src + "\n}"
```

Comment:
This is the bug fix - any comment at the end of the `src` was eating the closing `}`